Changed the runtime exception using RxJava

piyush0 opened the pull request:
Displayed repositories of hasgeek

karthikb351 commented:
Hi Piyush,

The expectation was that you would just fill in the code in the required places. Leaving 'RxJava' and the UI as is.

The UI is intentionally plain, there is no bonus for rewriting it to use a CardView or switching to v24 :)

Could you update this PR to just do the minimum required?

karthikb351 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Piyush Ajmani](https://trello.com/c/BHRm7BUE/2-piyush-ajmani)

piyush0 commented:
```
Used RxJava as recommended and removed the card view. Please review again :)
```

piyush0 commented:
Hello Karthik,

I have changed the code and used RxJava. Also, I have removed card view and
changed the layout back.

And this trello card is not loading.

On Wed, Oct 12, 2016 at 11:30 AM, Karthik Balakrishnan <
notifications@github.com> wrote:

> https://camo.githubusercontent.com/193766a3b9959c5f4ed5cd8cf3251d015e839d51/68747470733a2f2f6769746875622e7472656c6c6f2e73657276696365732f696d616765732f6d696e692d7472656c6c6f2d69636f6e2e706e67 Piyush
> Ajmani https://trello.com/c/BHRm7BUE/2-piyush-ajmani
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/hasgeek/internship-test-android/pull/5#issuecomment-253126714,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AT-9x2hqBGMvzo0DgXec2isrnWpTSOsLks5qzHeHgaJpZM4KT3_F
> .
